src/common/ui/code.vue

```diff
@@ -98,6 +102,24 @@ CodeMirror.commands.insertTab = cm => (
   cm.options.indentWithTabs ? insertTab(cm) : insertSoftTab(cm)
 );
 
+function autoHintWithFallback(cm, options) {
```

Comment:
Please use ESLint.

* `options` is already declared in the global scope so use `opts` or `cfg` or `config` and so on.
* we don't use `var`, we use `let`
* there are two spaces

It's also possible to rewrite the entire function using `?.`:
```js
CodeMirror.registerHelper('hint', 'autoHintWithFallback', (cm, opts) => {
  const result = cm.getHelper(cm.getCursor(), 'hint')?.(cm, opts);
  return result?.list.length ? result : CodeMirror.hint.anyword(cm, opts);
});
```

Reply:
Thanks! I didn't realize pre-push lint check was bypassed. And I didn't know `?.` is available.